index.html

```diff
@@ -47,5 +47,332 @@ <h1 class="headline-1-h1">I’m Mariane
+                    <div class="works-4-div2-1-b">
+                        <ul class="works-4-div2-1-b-ul">
+                            <li class="works-4-div2-1-b-ul-li-1">
+                                <span class="works-4-div2-1-b-ul-li-1-sp">CANOPY</span>
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-2">
+                                <img class="works-4-div2-1-b-ul-li-2-img" src="images/Counter.svg" alt="">
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-3">
+                                <span class="works-4-div2-1-b-ul-li-3-sp">Back End Dev</span>
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-4">
+                                <img class="works-4-div2-1-b-ul-li-4-img" src="images/Counter.svg" alt="">
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-5">
+                                <span class="works-4-div2-1-b-ul-li-5-sp">2015</span>
+                            </li>
+                        </ul>
+                </div>
+                <div class="works-4-div2-2">
+                    <p class="works-4-div2-2-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="works-4-div2-3">
+                    <button class="works-4-div2-3-btn1" type="button">
+                        <span class="works-4-div2-3-btn1-sp1">html</span>
+                    </button>
+                    <button class="works-4-div2-3-btn2" type="button">
+                        <span class="works-4-div2-3-btn2-sp2">css</span>
+                    </button>
+                    <button class="works-4-div2-3-btn3" type="button">
+                        <span class="works-4-div2-3-btn3-sp3">javaScript</span>
+                    </button>
+                </div>
+                <div class="works-4-div2-4">
+                    <button class="works-4-div2-4-btn" type="button">
+                        <span class="works-4-div2-4-btn-sp">See Project</span>
+                    </button>
+                </div>
+            </div>
+        </section>
+    </section>
+    <!--About Me-->
+    <section class="aboutme">
+        <div class="top">
+            <div class="top1">
+                <p class="top1-p">About<br>Myself</p>
+            </div>
+            <div class="top2">
+                <p class="top2-p">Hello I’m a software developer! I can help you 
+                    build a product , feature or website Look through some of my 
+                    work and experience! If you like what you see and have a project 
+                    you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="top3">
+                <div class="top3-1">
+                    <p class="top3-1-p">LET’S CONNECT</p>
```

Comment:
- [ ] same with the `LET’S CONNECT` title, it should be (h2,h3,etc...) tag , kindly fix it to be semantically correct  🙏